src/utils/common-expression-helpers.ts

```diff
@@ -468,3 +469,17 @@ export function registerDependency(node: FormNode, determinant: OHRIFormField) {
       determinant.fieldDependants.add(node.value['id']);
   }
 }
+
+type ConceptValue = typeof ConceptTrue | typeof ConceptFalse;
+
+export const booleanConceptToBoolean = (booleanConceptRepresentation: ConceptValue): Boolean => {
+  if (!booleanConceptRepresentation) {
+    throw new Error('booleanConceptRepresentation cannot be a null value');
+  }
+  if (booleanConceptRepresentation == ConceptTrue) {
+    return true;
+  }
+  if (booleanConceptRepresentation == ConceptFalse) {
+    return false;
+  }
+};
```

Comment:
There are two issues here:
1. This seems to be out of scope and ideally should be addressed in a follow-up PR
2. This seems like an Implementation detail and if we have to support this, it should be driven through a configuration.

Reply:
Out of curiosity, what is the rationale for this? Is it that there could be multiple concepts for true/ false?

Reply:
My point is if we are to support multiple implementations, we should not hardcode refs to non-system level metadata. In short, the `ConceptTrue` vs `ConceptFalse` may vary across downstream distros.

Reply:
Maybe it's a good idea to register this helper via the registry?

Reply:
@samuelmale I agree this is out of the scope of this PR. But I thought the conceptTrue / conceptFalse issue was a common problem... and also I don't see any conceptUUIDs here @arodidev how does this comparison really works?

Reply:
@eudson the function compares the argument passed to it with conceptTrue and conceptFalse which are hard-coded concept UUIDs for true/ false.